pkg/reconciler/inmemorychannel/dispatcher/inmemorychannel.go

```diff
@@ -26,19 +26,21 @@ import (
 
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
+
 	"knative.dev/eventing/pkg/kncloudevents"
```

Comment:
Same comment, `"knative.dev/eventing/pkg/kncloudevents"` is lonely and belongs around line 38 with other Kn packages.